client/__mocks__/is-my-json-valid.js

```diff
@@ -0,0 +1,40 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import jsonSchema from './lib/schema.json';
+import imjv from 'is-my-json-valid';
+import { forEach, get, isEmpty } from 'lodash';
+
+const validateSchema = imjv( jsonSchema, { verbose: true, greedy: true } );
```

Comment:
It would be nice to squeeze "draft4" in here somewhere, maybe this variable name, in an attempt to save future devs that particular confusion.

Reply:
👍  00d76c0